pkg/sensors/platform/source/acpi.go

```diff
@@ -109,12 +107,14 @@ func (a *ACPI) StopPower() {
 }
 
 func (a *ACPI) GetCPUCoreFrequency() map[int32]uint64 {
+	freqPathDir := config.SysDir() + "/devices/system/cpu/cpufreq/"
 	files, err := os.ReadDir(freqPathDir)
 	cpuCoreFrequency := map[int32]uint64{}
 	if err != nil {
 		klog.Warning(err)
 		return cpuCoreFrequency
 	}
+	freqPath := config.SysDir() + "/devices/system/cpu/cpufreq/policy%d/scaling_cur_freq"
```

Comment:
keep the `/devices/system/cpu/cpufreq/policy%d/scaling_cur_freq` as constants and use the constants here.

Reply:
same comment for other places.